### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by weijia-cheng** [lukasbystricky/john-dewey_human-nature-and-conduct#1]
# Semantics

- [ ] Assuming that “J. D.” in the preface stands for John Dewey, it should probably be marked as a `z3998:personal-name` instead of `z3998:given-name`.
- [ ] I think these can be deitalicized due to being in Merriam-Webster:

- élan
- élan vital
- sine qua non

# Commit History

- [ ] fc640363375e3bdee27adf91a959c6f45b831218 has a typo in the commit message.
- [ ] 9d837dcc5c3574ff0c66d7fe250d0e9ad9912480 should probably say “Southsea” instead of “southsea.”
- [ ] 94459272ddca0d86aea75dbc5c16457b3a080164: I feel like inserting a word is a much more significant editorial change than just changing a punctuation mark or spelling (what a lot of printer’s typo fixes are). Of the changes you made, I would revert the following:

- `that every attained satisfaction is [-only -]forever bound`
- `a thing to be decided by {+a +}scientifically conducted study of consequences`
- `Difficulty in reading the stamp is due to {+the +}variety of impressions`

The last two are unusual usages, but I don’t think there is anything wrong about them, especially if the source material was a lecture, since omitting articles might be a way of representing the terseness/tone of the original material. The first one seems like it could be a major change in the meaning of the text, even though I agree that the printed text looks odd. Did you have a particular justification for this change?

**Comment by lukasbystricky** [comment]
Thanks for the feedback Weijia. 

I was a bit more aggressive than I normally would be with the printer errors simply because the PG transcriber had already made quite a few edits, some of which were typos (I think some of my modernizations fell into that category too), but others were clear cases of missing words. It was originally a lecture, but the book was not meant to be a transcription. Dewey spent several years putting it into a book form. So my guess is that this was just a somewhat sloppy printing. 

For the first one it originally read "...  that every attained satisfaction is only forever bound to be only a disappointment ...," which seemed to me like it had a superfluous "only". It's definitely a mistake I've made in my own writing, but yeah maybe that's me over-correcting. For the last two, don't "scientifically conducted" and "variety" need articles in front of them?

**Comment by weijia-cheng** [comment]
Well, for #2, it feels fairly clear to me that the article is not required. Adding the "a" makes it an indefinite article, suggesting that there some nonspecific study exists that would be able to decide the question. This would be usage 2d(1) of [study](https://www.merriam-webster.com/dictionary/study) in Merriam-Webster. But by omitting the article, I think that you lean toward something like usage 2b, which is just talking about a general field of investigation. I feel like that there's a potential change in meaning going on here with your addition which is why I suggested against it.

For #3, I think [ngrams](https://books.google.com/ngrams/graph?content=due+to+variety+of%2Cdue+to+the+variety+of&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3) is a good illustration that "due to variety of" is an idiomatic (if less common) usage, especially if you look at the usages in Google Books. Hence I think you should leave it, especially since I don't think there's any significant readability gain in this change.

**Comment by lukasbystricky** [comment]
Yes, of course you're right, #2 is completely fine. I must have been stuck in over-correct mode. #3 still sounds wrong to me, but I suppose it could be idiomatic as you say. 

I've reverted all those changes. I think everything else has been addressed too. Thanks again!
